dotnet/src/IntegrationTests/Fakes/EmailSkillFake.cs

```diff
@@ -20,8 +20,8 @@ internal sealed class EmailSkillFake
 
     [SKFunction, Description("Lookup an email address for a person given a name")]
     public Task<string> GetEmailAddressAsync(
-        [Description("The name of the person to email.")] string input,
-        ILogger logger)
+        ILogger logger,
+        [Description("The name of the person to email.")] string input = "")
```

Comment:
To maintain the logic in the method:
```
        if (string.IsNullOrEmpty(input))
        {
            logger.LogTrace("Returning hard coded email for {0}", input);
            return Task.FromResult("johndoe1234@example.com");
        }
```
If we want to allow no value to be passed, then we have to give it a default, as the property mapping uses defaults indicate optional parameters. `null` would fail to resolve from the dictionary and throw.
I'll update it to `string? input = null;` so that it's still optional, but not automatically empty.